neuromation/cli/admin.py

```diff
@@ -45,12 +49,121 @@ def admin() -> None:
         )
 
 
+@command()
+@click.argument(
+    "config",
+    required=False,
+    type=click.Path(exists=False, path_type=str),
+    default="cluster.yml",
+)
+@click.option("--type", prompt="Select cluster type", type=click.Choice(["aws", "gcp"]))
+@async_cmd()
+async def generate_cluster_config(root: Root, config: str, type: str) -> None:
+    """
+    Create a cluster configuration file.
+    """
+    config_path = pathlib.Path(config)
+    if config_path.exists():
+        raise ValueError(
+            f"Config path {config_path} already exists, "
+            "please remove the file or pass the new file name explicitly."
+        )
+    if type == "aws":
+        content = await generate_aws()
+    elif type == "gcp":
+        content = await generate_gcp()
+    else:
+        assert False, "Prompt should prevent this case"
+    config_path.write_text(content)
+    if not root.quiet:
+        click.echo(f"Cluster config {config_path} is generated.")
+
+
+AWS_TEMPLATE = """\
+type: aws
+region: us-east-1
+zones:
+- us-east-1a
+- us-east-1b
+vpc_id: {vpc_id}
+credentials:
+  access_key_id: {access_key_id}
+  secret_access_key: {secret_access_key}
+node_pools:
+- id: m5_2xlarge
+  min_size: 1
+  max_size: 4
+- id: p2_xlarge_1x_nvidia_tesla_k80
+  min_size: 1
+  max_size: 4
+- id: p3_2xlarge_1x_nvidia_tesla_v100
+  min_size: 0
+  max_size: 1
+storage:
+  id: generalpurpose_bursting
+"""
+
+
+async def generate_aws() -> str:
+    aws_config_file = pathlib.Path(
+        os.environ.get("AWS_CONFIG_FILE", "~/.aws/credentials")
+    )
+    aws_config_file = aws_config_file.expanduser().absolute()
+    parser = configparser.ConfigParser()
+    parser.read(aws_config_file)
+    args = {}
+    args["vpc_id"] = click.prompt("AWS VPC ID")
+    profile = click.prompt("AWS profile name", default="default")
```

Comment:
might be out of scope. should we ask a user whether they want to use the shared credentials file, or rather infer the credentials from the corresponding env vars, or even prompt for them?

also related to the docs @mariyadavydova

Reply:
Thanks for the suggestion.
I've updated the PR to respect more AWS env vars.